src/transmuxer-worker.js

```diff
@@ -307,6 +307,10 @@ class MessageHandlers {
     this.transmuxer.setAudioAppendStart(Math.ceil(secondsToVideoTs(data.appendStart)));
   }
 
+  setRemux(data) {
```

Comment:
Remux basically means that `mux.js` should output one transmuxed segment for `audio` and `video` data combined.